lib/stringutils/filter_parser.go

```diff
@@ -1,45 +1,49 @@
-package params
+package stringutils
 
 import (
-	"fmt"
+	"errors"
 	"net/url"
 	"strings"
-
-	"github.com/pkg/errors"
 )
 
 // FormatFilters Will receive an array of filters and will format them into a map of strings
 //
 // Example:
 //   [
-//    "status:critical",
-//    "service_name:redis",
-//    "service_name:postgres",
-//    "application:cafe",
-//    "environment:prod",
+//    "environment:adios",
+//    "environment:hola",
+//    "cookbook:awesome",
+//    "roles:lalala",
 //   ]
 //
 // The returned filters would look like:
 //
 // map[string][]string [
-// 	"status": ["critical"],
-// 	"service_name": ["redis","postgres"],
-// 	"application": ["cafe"],
-// 	"environment": ["prod"]
+// 	"environment": ["adios","hola"],
+// 	"cookbook": ["awesome"],
+// 	"roles": ["lalala"],
 // ]
-// TODO: (afiune) Migrate this to a common go package that other teams can consume
```

Comment:
I see what you did there! Super nice Lance 💯

![tenor-245281441](https://user-images.githubusercontent.com/5712253/62957971-1bf29d00-bdf6-11e9-9a71-6ee5d5d7733d.gif)